scripts/stage-release.sh

```diff
@@ -111,50 +122,32 @@ fi
 
 if [[ "${SKIP_VALIDATION:-false}" != true ]]; then
     printf "Validating the build is ${GREEN}green${NC}"
-    mvn -q clean verify
+    mvn -q clean verify -DprocessAllModules=true
 fi
 
-mvn -q versions:set -DnewVersion="${RELEASE_VERSION}" -DgenerateBackupPoms=false -DprocessAllModules=true
+echo "Versioning Kroxylicious as ${RELEASE_VERSION}"
 
+mvn -q versions:set -DnewVersion="${RELEASE_VERSION}" -DgenerateBackupPoms=false -DprocessAllModules=true
 echo "Validating things still build"
 mvn -q clean install -Pquick
 
 
 RELEASE_TAG="v${RELEASE_VERSION}"
 
-echo "Committing release to git"
+echo "Committing framework release to git"
 git add '**/pom.xml' 'pom.xml'
 git commit --message "Release version ${RELEASE_TAG}" --signoff
 
 git tag -f "${RELEASE_TAG}"
 
 git push "${REPOSITORY}" "${RELEASE_TAG}" ${GIT_DRYRUN:-}
 
-echo "Deploying release to maven central"
-mvn deploy -Prelease -DskipTests=true -DreleaseSigningKey="${GPG_KEY}" -DprocessAllModules=true ${MVN_DEPLOY_DRYRUN:-}
-
-echo "Creating binary distribution"
-mvn package -Pdist -DskipTests=true
-
-if ! command -v gh &> /dev/null
-then
-    echo "gh command could not be found. Please create a release by hand https://github.com/kroxylicious/kroxylicious/releases/new"
-else
-  echo "Creating GitHub release with binary distribution"
-  ORIGINAL_GH_DEFAULT_REPO=$(gh repo set-default -v | (grep -v 'no default repository' || true))
-  gh repo set-default $(git remote get-url ${REPOSITORY})
-  # create GitHub release via CLI https://cli.github.com/manual/gh_release_create
-  if [[ "${DRY_RUN:-false}" == true ]]; then
-    gh release create "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md" --draft
-    echo "Draft release created. This must be manually published."
-  else
-    gh release create "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md"
-  fi
-fi
+echo "Deploying release"
+mvn -q deploy -Prelease,dist -DskipTests=true -DreleaseSigningKey="${GPG_KEY}" ${MVN_DEPLOY_DRYRUN} -DprocessAllModules=true
```

Comment:
@gracegrimwood I tested out adding `dist` profile.  The tgz gets signed and pushed to the staging repo okay.